src/test/ui/resolve/levenshtein.stderr

```diff
@@ -2,19 +2,28 @@ error[E0412]: cannot find type `esize` in this scope
   --> $DIR/levenshtein.rs:15:11
    |
 15 | fn foo(c: esize) {} // Misspelled primitive type name.
-   |           ^^^^^ did you mean `isize`?
+   |           ^^^^^
+   |           |
+   |           not found in this scope
+   |           did you mean `isize`?
 
 error[E0412]: cannot find type `Baz` in this scope
   --> $DIR/levenshtein.rs:19:10
    |
 19 | type A = Baz; // Misspelled type name.
-   |          ^^^ did you mean `Bar`?
+   |          ^^^
+   |          |
+   |          not found in this scope
```

Comment:
Hi @petrochenkov, could you clarify a bit on how @Freyskeyd would fix this so that fall-back labels aren't reported?